text/chapter5.md

```diff
@@ -444,12 +449,18 @@ current :: Number
 current = calculateCurrent lightbulb lightbulb -- uncaught mistake
 ```
 
-Note that the constructor of a newtype often has the same name as the newtype itself, but this is not a requirement. For example, unique names are also valid:
+Note that while a newtype can only have a single constructor, and the constructor must be of a single value, a newtype _can_ take any number of type variables. For example, the following newtype would be a valid definition:
+
+```Haskell
+newtype CouldError err a = CouldError (Either err a)
+```
```

Comment:
This confused me a lot when first learning newtypes.  Maybe it's not a common enough misconception to be worth pointing out, but I thought I'd add it for consideration.